package.json

```diff
@@ -68,6 +69,9 @@
         "jest",
         "sku"
       ]
+    },
+    "overrides": {
+      "tsm>esbuild": "^0.18.10"
```

Comment:
Why the need for the override here?

Reply:
Keep in mind this only affects the monorepo. I was debugging `crackle dev` and needed a newer version of esbuild. This was the recommended workaround https://github.com/lukeed/tsm/issues/48#issuecomment-1610005975